### DISCUSSION TITLE
请问如何设置查询结果导航面板中图标的大小

### DISCUSSION THREAD

**Discussion by Pseud1nymer** [xiaoyifang/goldendict-ng#1604] [Q&A]
目前的查询结果导航面板中的图标太大了，想设置成欧路的效果，请问如何设置？是在article-style.css中吗？
![14 16 44](https://github.com/xiaoyifang/goldendict-ng/assets/173440959/b293f51c-9421-4ede-aabb-a70d7e1450bd)

**Comment by zqf2014** [comment]
太大了，应该提供一个选择，可根据喜好自行决定大图标还是小图标
